base_comment_template/models/base_comment_template.py

```diff
@@ -124,37 +101,29 @@ class BaseCommentTemplate(models.Model):
         help="This comment template will be available on this models. "
         "You can see here only models allowed to set the coment template.",
     )
-
     domain = fields.Char(
-        "Filter Domain",
+        string="Filter Domain",
         required=True,
         default="[]",
         help="This comment template will be available only for objects "
         "that satisfy the condition",
     )
-
-    priority = fields.Integer(
-        default=10, copy=False, help="the highest priority = the smallest number",
+    sequence = fields.Integer(
+        required=True, default=1, help="The smaller number = The higher priority"
```

Comment:
Ten is a good default and follows previous value.
```suggestion
        required=True, default=10, help="The smaller number = The higher priority"
```